### DISCUSSION TITLE
Sfdx-Hardis github instructions

### DISCUSSION THREAD

**Discussion by Heartist3** [hardisgroupcom/sfdx-hardis#522] [Q&A] [answered]
Good morning to the community. Recently I decided that I need to change my workflow with SF products (my background is not developer but I try my best to learn new skills) and reviewed a lot of solutions (free source and payed ones) while learning how to use git and sfdx.

One of the products I am researching currently is sfdx-hardis and going through the documentation I can say that is not difficult to understand especially if you utilise Gitlab and you have a solid understanding of what is going on. But for people like me (gaining knowledge and using github) I believe a github "dedicated" instruction set geared towards the not so savvy users would be beneficial. 

Questions within a question 
-> do you believe it would be possible to "combine" sfdx-hardis with salesforce DevOps center?

DevOps center for the none developer savvy and sfdx-hardis for the people who wish to expand their skills

Understandably sfdx-hardis provides the user interface to execute all that is needed but "establishing" it within a company us ours that preaches technology but is fearful of changes would be a long shot... Establishing it though in combination with salesforce DevOps center would allow a more seamless transition to the full adoption of the product.

Thank you in advance for any assistance provided.

**Comment by nvuillam** [comment]
Hi @Heartist3 , 

Thanks for your interest in sfdx-hardis :)
I look closely at what happens around DevOps center, but currently with the impossibility to override workflows (DoC works with direct calls to GitHub API), + the bad feedbacks I receive from everyone who has used it... it is currently not on the RoadMap to integration DoC with sfdx-hardis, for the simple reason that I have no clue about what could be done and what it would bring to sfdx-hardis :D 

But the good news is that sfdx-hardis is perfectly compliant with GitHub since @legetz created the GitHub Actions workflows, and the even better news is that he and others provided feedback that many Salesforce production orgs are fed with sfdx-hardis + Github

So basically, you can just go for sfdx-hardis + GitHub , it works and it has been proved ;)

**Reply by Heartist3** [reply]
@nvuillam Thank you for your prompt reply. 

My question was geared more towards to if the community (people who are using it - and have a better understanding of the processes - which I currently do not have) believes that it is possible to have a workflow - not a direct integration - (one that may experience disturbances) where none admins will keep on working via devOps center and more technical people will be utilising sfdx-hardis?

Will investigate further the documentation to locate the GitHub actions workflow from @legetz and go through the material.

ps: I am "spamming" reddit with the documentation of the product. With the little I understand so far, sfdx-hardis is brilliant.

Thank you

**Reply by nvuillam** [reply, marked as answer]
@Heartist3 just follow the setup guide and at the end you'll have your working GitHub workflows :) (they are copied in your repo when initializing hardis config ^^)

https://sfdx-hardis.cloudity.com/salesforce-ci-cd-setup-home/

If you have issues., post them here, we'll help :)
